openedx/core/djangoapps/discussions/tests/test_admin.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.test import Client
+from openedx.core.djangoapps.discussions.models import DiscussionsConfiguration
+
+
+class DiscussionsConfigurationAdminTest(TestCase):
+    """
+    Tests for discussion config admin
+    """
+    def setUp(self):
+        super().setUp()
+        self.superuser = User.objects.create_superuser('admin', 'admin@example.com', 'password')
+        self.client = Client()
+        self.client.login(username='admin', password='password')
```

Comment:
```suggestion
        self.client.login(username=self.superuser.username, password='Password1234')
```

The default password for all factory users is `Password1234`.